.github/actions/templates/avm-validateModuleDeployment/action.yml

```diff
@@ -55,14 +55,47 @@ inputs:
   customLocation:
     description: "Custom location overwrite, if needed"
     required: false
+  e2eIgnore:
+    description: "A test case will not be deployed if an .e2eignore file exists"
+    required: false
 
 runs:
   using: "composite"
   steps:
+    # [Deployment validation] task(s)
+    # -------------------------------
+    - name: "Validate Test Execution"
+      uses: azure/powershell@v2
+      with:
+        azPSVersion: "latest"
+        errorActionPreference: "SilentlyContinue"
+        inlineScript: |
+          # the first step is to check, if the deployment should be bypassed
+
+          # Resolve template file path
+          $moduleTestFilePath = Join-Path $env:GITHUB_WORKSPACE '${{ inputs.templateFilePath }}'
+
+          # Check if the deployment test should be bypassed
+          if ('${{ inputs.e2eIgnore }}' -eq 'true') {
+            Write-Warning -Message "File '.e2eignore' exists in the folder: $moduleTestFilePath"
+            Write-Output 'skip_deployment_ci=true' >> $env:GITHUB_ENV
+
+            # get the reason for skipping the deployment
+            $excludeReason = Get-Content Join-Path $moduleTestFilePath '.e2eignore'
```

Comment:
Does this concatanate strings?

Would have the same question for this line: `$jobOutput = Get-Content $mdPesterOutputFilePath '\n' $validateTestExecutionResult`

Just asking as usually everything after a function call (in this case `Get-Content`) is a parameter (a 'positional parameter' to be precise). So this reads like `\n` &  `$validateTestExecutionResult` are parameters of the function, though I guess they're actually string concatinations as they don't make sense as parameters. 

Let me know, as I'm quite confused right now 😄